### PR TITLE
Conclude `Skinning Contest #1` and add the index page

### PR DESCRIPTION
**Pull request by zeusminus**
This PR...
- specifies that this has been the first iteration of the Skinning Contest
- creates an index file and moves the files related to the first iteration inside the `1` folder
- adds a link to the [Results Out newspost](https://osu.ppy.sh/home/news/2021-05-27-skinning-contest-results)
- adds the podium image and updates prizing details accordingly

### CODE REVIEW

File: wiki/Contests/Skinning_Contest/1/en.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
+---
+tags:
+  - skinning
+  - skinship
+  - skins
+---
+
+# Skinning Contest #1
+
+![Skinning Contest #1 Banner](img/banner.jpg)
```
Comment (TPGPL) [inline wiki/Contests/Skinning_Contest/1/en.md:10]:
```suggestion
![Skinning Contest #1 banner](img/banner.jpg)
```

File: wiki/Contests/Skinning_Contest/1/en.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+The **Skinning Contest #1** is a free-for-all osu! skinning contest hosted by [skinship](https://skinship.xyz/ "skinship"), one of the largest skinning communities on osu!. This is the first official osu! skinning contest, and the initial iteration of Skinning Contests hosted by skinship.
```
Comment (TPGPL) [inline wiki/Contests/Skinning_Contest/1/en.md:12]:
```suggestion
The **Skinning Contest #1** was a free-for-all osu! skinning contest hosted by [skinship](https://skinship.xyz "skinship"), one of the largest skinning communities on osu!. It was the first official osu! skinning contest, and the initial iteration of Skinning Contests hosted by skinship.
```

File: wiki/Contests/Skinning_Contest/1/en.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 6 months of osu!supporter, unique profile badge |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 4 months of osu!supporter, unique profile badge |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 2 months of osu! supporter, unique profile badge |
+
+## Organisation
+
+The Skinning Contest is run by various community members.
```
Comment (TPGPL) [inline wiki/Contests/Skinning_Contest/1/en.md:33]:
```suggestion
The Skinning Contest #1 was run by various community members.
```

File: wiki/Contests/Skinning_Contest/1/en.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+| Member | Responsibility |
```
Comment (TPGPL) [inline wiki/Contests/Skinning_Contest/1/en.md:35]:
```suggestion
| User | Responsibility |
```

File: wiki/Contests/Skinning_Contest/1/en.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+| :-- | :-- |
+| ![][flag_DE] [Master-TigerKun](https://osu.ppy.sh/users/10688456) | Contest planning |
+| ![][flag_RO] [Matt2e2](https://osu.ppy.sh/users/12144912) | Contest planning, graphic design |
+| ![][flag_NL] [Roan](https://osu.ppy.sh/users/8214639) | Contest planning |
+| ![][flag_DE] [RockRoller](https://osu.ppy.sh/users/8388854) | Contest planning, submission screening, showcase preparations |
+| ![][flag_GB] [tetsui](https://osu.ppy.sh/users/10974678) | Contest planning, showcase preparations |
+| ![][flag_PL] [watterino](https://osu.ppy.sh/users/3512261) | Video editor, showcase preparations |
+| ![][flag_NL] [vvivi](https://osu.ppy.sh/users/10432755) | Contest planning |
```
Comment (TPGPL) [inline wiki/Contests/Skinning_Contest/1/en.md:43]:
```suggestion
| ![][flag_NL] [vvivi](https://osu.ppy.sh/users/10432755) | Contest planning |
| ![][flag_PL] [watterino](https://osu.ppy.sh/users/3512261) | Video editor, showcase preparations |
```

File: wiki/Contests/Skinning_Contest/1/en.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+
+## Links
+
+- [**Contest page**](https://osu.ppy.sh/community/contests/126)
```
Comment (TPGPL) [inline wiki/Contests/Skinning_Contest/1/en.md:47]:
```suggestion
- **[Contest page](https://osu.ppy.sh/community/contests/126)**
```

File: wiki/Contests/Skinning_Contest/1/en.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+- [Announcement newspost](https://osu.ppy.sh/home/news/2021-03-08-skinning-contest-announcement)
+- [Voting Open newspost](https://osu.ppy.sh/home/news/2021-04-29-skinning-contest-voting-open)
+- [Results Out newspost](https://osu.ppy.sh/home/news/2021-05-27-skinning-contest-results)
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1266342)
+- [Submission thread](https://osu.ppy.sh/community/forums/topics/1266324)
+- [Discord server](https://discord.skinship.xyz/)
+- [Official website](https://skinship.xyz/)
```
Comment (TPGPL) [inline wiki/Contests/Skinning_Contest/1/en.md:54]:
```suggestion
- [Discord server](https://discord.gg/q8Z5JXM "Discord")
- [Official website](https://skinship.xyz "skinship")
```

File: wiki/Contests/Skinning_Contest/1/en.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+| ![][flag_VN] [TKieen](https://osu.ppy.sh/users/12561202) | [Randoline](https://osu.ppy.sh/community/forums/topics/1291235) |
+| ![][flag_MX] [xet](https://osu.ppy.sh/users/16274977) | [Who Tao](https://osu.ppy.sh/community/forums/topics/1281532) |
+| ![][flag_PH] [Xyber Voltage](https://osu.ppy.sh/users/10443089) | [FreeMod is EZ](https://osu.ppy.sh/community/forums/topics/1269099) |
+| ![][flag_MX] [ZixkyST](https://osu.ppy.sh/users/11844975) | [ZixStar](https://osu.ppy.sh/community/forums/topics/1278132) |
+
+## Podium
+
+*For full results, see the [contest page](https://osu.ppy.sh/community/contests/126).*
+
+![Skinning Contest #1 Podium](img/podium.png)
```
Comment (TPGPL) [inline wiki/Contests/Skinning_Contest/1/en.md:96]:
```suggestion
![](img/podium.png "Skinning Contest #1 podium")
```

File: wiki/Contests/Skinning_Contest/1/en.md [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+
+## Rules
```
Comment (TPGPL) [inline wiki/Contests/Skinning_Contest/1/en.md:98]:
```suggestion
## Ruleset
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Comment by TPGPL** [comment]
also the podium needs to be compressed, you can use [this one](https://tinypng.com/web/output/3vebu03x08vwzx2wbwb93beb3f64npp6/f.png)

File: wiki/Contests/Skinning_Contest/1/en.md [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+The **Skinning Contest #1** was a free-for-all osu! skinning contest hosted by [skinship](https://skinship.xyz "skinship"), one of the largest skinning communities on osu!. It was the first official osu! skinning contest, and the initial iteration of Skinning Contests hosted by skinship.
+
+## Contest schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Announcement | 2021-03-08 |
+| Submission phase | 2021-03-08/2021-04-08 |
+| Voting phase | 2021-04-29/2021-05-13 |
+| Results | 2021-05-20 |
```
Comment (RockRoller01) [inline wiki/Contests/Skinning_Contest/1/en.md:21]: This is incorrect, results came out a week late. The correct date is 2021-05-27

File: wiki/Contests/Skinning_Contest/1/en.md [hunk 11]
```diff
@@ -0,0 +1,139 @@
...
+| ![][flag_UA] [Pirasto](https://osu.ppy.sh/users/6766278) | [Prinz Heinrich](https://osu.ppy.sh/community/forums/topics/1292847) |
+| ![][flag_AU] [spiefie99](https://osu.ppy.sh/users/10934017) | [Glassmorphism](https://osu.ppy.sh/community/forums/topics/1269582) |
+| ![][flag_VE] [Suminoze](https://osu.ppy.sh/users/5308780) | [1stTry](https://osu.ppy.sh/community/forums/topics/1285565) |
+| ![][flag_PL] [tearek\_](https://osu.ppy.sh/users/20767121) | [Multicolor Diamond HUD](https://osu.ppy.sh/community/forums/topics/1272664) |
+| ![][flag_VN] [TKieen](https://osu.ppy.sh/users/12561202) | [Randoline](https://osu.ppy.sh/community/forums/topics/1291235) |
+| ![][flag_MX] [xet](https://osu.ppy.sh/users/16274977) | [Who Tao](https://osu.ppy.sh/community/forums/topics/1281532) |
+| ![][flag_PH] [Xyber Voltage](https://osu.ppy.sh/users/10443089) | [FreeMod is EZ](https://osu.ppy.sh/community/forums/topics/1269099) |
+| ![][flag_MX] [ZixkyST](https://osu.ppy.sh/users/11844975) | [ZixStar](https://osu.ppy.sh/community/forums/topics/1278132) |
+
+## Podium
```
Comment (RockRoller01) [inline wiki/Contests/Skinning_Contest/1/en.md:92]: Should maybe link the winners as there is no easy way to get to them, as their names are on the image only currently.

Maybe would also be nice to show the 3 badges in this section as well?

**Review by RockRoller01 — CHANGES_REQUESTED** [review]

File: wiki/Contests/Skinning_Contest/1/en.md (hunk 7 above)
Comment (RockRoller01) [reply wiki/Contests/Skinning_Contest/1/en.md]: Don't see the point of this one? a domain based invite is more likely to survive over time than a direct one. The discord one may change but the domain one will always lead to the server

File: wiki/Contests/Skinning_Contest/1/en.md [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+  - skinning
+  - skinship
+  - skins
+---
+
+# Skinning Contest #1
+
+![Skinning Contest #1 banner](img/banner.jpg)
+
+The **Skinning Contest #1** was a free-for-all osu! skinning contest hosted by [skinship](https://skinship.xyz "skinship"), one of the largest skinning communities on osu!. It was the first official osu! skinning contest, and the initial iteration of Skinning Contests hosted by skinship.
```
Comment (Walavouchey) [inline wiki/Contests/Skinning_Contest/1/en.md:12]: consistency

```suggestion
The **Skinning Contest #1** was a free-for-all osu! skinning contest hosted by [skinship](https://skinship.xyz "skinship"), one of the largest skinning communities on osu!. It was the first official osu! skinning contest, and the first iteration of Skinning Contests hosted by skinship.
```

File: wiki/Contests/Skinning_Contest/1/en.md [hunk 13]
```diff
@@ -0,0 +1,141 @@
...
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 6 months of osu!supporter, unique profile badge |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 4 months of osu!supporter, unique profile badge |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 2 months of osu! supporter, unique profile badge |
+
+![Winner profile badge](img/badge-winner.png "Skinning Contest #1 Winner") ![2nd place profile badge](img/badge-2nd.png "Skinning Contest #1 2nd place") ![3rd place profile badge](img/badge-3rd.png "Skinning Contest #1 3rd place")
```
Comment (TPGPL) [inline wiki/Contests/Skinning_Contest/1/en.md:31]:
```suggestion
![](img/badge-winner.png "Skinning Contest #1 winner badge") ![](img/badge-2nd.png "Skinning Contest #1 2nd place badge") ![](img/badge-3rd.png "Skinning Contest #1 3rd place badge")
```

File: wiki/Contests/Skinning_Contest/1/en.md [hunk 14]
```diff
@@ -0,0 +1,141 @@
...
+
+## Links
+
+- **[Contest page](https://osu.ppy.sh/community/contests/126)**
+- [Announcement newspost](https://osu.ppy.sh/home/news/2021-03-08-skinning-contest-announcement)
+- [Voting Open newspost](https://osu.ppy.sh/home/news/2021-04-29-skinning-contest-voting-open)
+- [Results Out newspost](https://osu.ppy.sh/home/news/2021-05-27-skinning-contest-results)
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1266342)
+- [Submission thread](https://osu.ppy.sh/community/forums/topics/1266324)
+- [Discord server](https://discord.gg/q8Z5JXM "Discord")
```
Comment (TPGPL) [inline wiki/Contests/Skinning_Contest/1/en.md:55]:
```suggestion
- [Discord server](https://discord.skinship.xyz "skinship")
```

File: wiki/Contests/Skinning_Contest/1/en.md [hunk 15]
```diff
@@ -0,0 +1,141 @@
...
+| Voting phase | 2021-04-29/2021-05-13 |
+| Results | 2021-05-27 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 6 months of osu!supporter, unique profile badge |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 4 months of osu!supporter, unique profile badge |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 2 months of osu! supporter, unique profile badge |
```
Comment (Walavouchey) [inline wiki/Contests/Skinning_Contest/1/en.md:29]:
```suggestion
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 2 months of osu!supporter, unique profile badge |
```

**Review by TPGPL — APPROVED** [review]
lgtm

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**